conf/mini_provisioner/v2/setup.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+ha:
+    post_install:
+        cmd: ha_setup post_install
+        args: --config <URL>

```

Comment:
Do we need to keep --config in args like this in mini provisioner ot it will be automatically added and we should remove args?
@nitin-seagate

Reply:
No response from @nitin-seagate. Let's resolve this during integration.